src/psqlgraph/base.py

```diff
@@ -276,29 +276,6 @@ def create_hybrid_properties(mapper, cls):
         cls.__pg_properties__[pg_attr] = f.__pg_types__
 
 
-class VoidedBaseClass:
-    @hybrid_property
-    def props(self):
-        """Alias of properties"""
-        return self.properties
-
-    @props.setter
-    def props(self, properties):
-        """Alias of properties"""
-        self.properties = properties
-
-    @hybrid_property
-    def sysan(self):
-        """Alias of properties"""
-        return self.system_annotations
-
-    @sysan.setter
-    def sysan(self, sysan):
-        """Alias of properties"""
-        self.system_annotations = sysan
-
-
-VoidedBase = declarative.declarative_base(cls=VoidedBaseClass)
```

Comment:
Here is a [search](https://github.com/search?q=org%3ANCI-GDC%20psqlgraph.base.VoidedBase&type=code) of the current usage. Not much, but once this gets into develop, these project might be broken

Reply:
@BenX00 , if you write up a ticket describing what needs to be done and prioritize it on the back log, then this can proceed.

Reply:
I've started capturing these things [here](https://gdc-ctds.atlassian.net/browse/DEV-3251). This will be simplified ultimately w/ calls to  `psqgraph.drop_all`